### PR TITLE
Archive component

### PR DESCRIPTION
**Pull request by frago12**
## What Changed?

Added Archive component to allow the download of deprecated docs

**Examples**
```jsx
<Archive
  title="Installation Guide"
  path="/docs/epas/9.6/EDB_Postgres_Advanced_Server_Installation_Guide_v9.6.pdf"
/>
```

### CODE REVIEW

File: src/components/archive.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const url = `https://github.com/EnterpriseDB/docs-archive/raw/${environment}${path}`;
+
+  return (
+    <Link to={url} title={title} className="w-100 d-block" {...props}>
+      <PdfIcon /> {title}
+    </Link>
+  );
+}
+
+const PdfIcon = ({ className }) => (
```
Comment (gsong) [inline src/components/archive.js:18]: We don't need any props, right?

File: src/components/archive.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Icon from "./icon";
+import Link from "./link";
+
+const environment = process.env.GATSBY_ENVIRONMENT_BRANCH;
+
+export default function Archive({ title, path, ...props }) {
+  const url = `https://github.com/EnterpriseDB/docs-archive/raw/${environment}${path}`;
```
Comment (gsong) [inline src/components/archive.js:9]: I would consider adding a leading slash to `path` if the user doesn't supply one.

**Review by gsong — APPROVED** [review]

File: src/components/archive.js (hunk 1 above)
Comment (frago12) [reply src/components/archive.js:18]: right!

File: src/components/archive.js (hunk 2 above)
Comment (frago12) [reply src/components/archive.js]: 👍

**Pull request merged**